From d811daefb3d686c4b6cd3f225988892c57c2d370 Mon Sep 17 00:00:00 2001
From: Raviteja Narayanam <raviteja.narayanam@xilinx.com>
Date: Mon, 9 Nov 2020 15:45:09 +0530
Subject: [PATCH 1752/1851] regulator: da9121: Use core regulator lock

commit 28e3c4894ca74ef9aef1698aea235bdab591437b from
https://github.com/Xilinx/linux-xlnx.git

Instead of directly accessing the mutex, use the regulator_lock/unlock
API provided by the core.

Signed-off-by: Raviteja Narayanam <raviteja.narayanam@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 drivers/regulator/da9121-regulator.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/regulator/da9121-regulator.c b/drivers/regulator/da9121-regulator.c
index 04560a1466a3..7757d2ba4d29 100644
--- a/drivers/regulator/da9121-regulator.c
+++ b/drivers/regulator/da9121-regulator.c
@@ -745,9 +745,9 @@ static inline int da9121_handle_notifier(
 			break;
 		}
 
-		mutex_lock(&rdev->mutex);
+		regulator_lock(rdev);
 		regulator_notifier_call_chain(rdev, notification, NULL);
-		mutex_unlock(&rdev->mutex);
+		regulator_unlock(rdev);
 	}
 
 error:
-- 
2.31.1

